tree/ntuple/v7/inc/ROOT/RNTupleModel.hxx

```diff
@@ -307,37 +326,67 @@ public:
    /// Creating projections for fields containing `std::variant` or fixed-size arrays is unsupported.
    RResult<void> AddProjectedField(std::unique_ptr<ROOT::RFieldBase> field, FieldMappingFunc_t mapping);
 
+   /// Transitions a RNTupleModel from the Building state to the Frozen state, disabling adding additional fields and enabling
+   /// creating entries from it.
+   /// Freezing an already-frozen model is a no-op.
+   /// Throws an RException if the model is in the Expired state.
+   /// See RNTupleModel for more explanations on the state transitions.
```

Comment:
```suggestion
   /// See RNTupleModel for more explanation on the state transitions.
```

Reply:
Is "explanation" uncountable? In which case, shouldn't we say "for a more detailed explanation" or something? ("more explanation" sounds wrong to me..)

Reply:
As far as I'm aware in this context it is, because we're explaining one single topic (state transtitions, it doesn't matter that there are multiple aspects to it). "More explanation" might be a Dutchism, so changing it to "for a more detailed explanation" sounds good to me as well.